### DISCUSSION TITLE
constantly loosing network drives

### DISCUSSION THREAD

**Discussion by lordtheobald** [Chia-Network/chia-blockchain#7786] [Support]
I am trying to find out what is going on. I have multiple computers and I am using them all to create plots that are saved to a network drive, but one computer Chia constantly loses the directories. The directories are in the config file and even closing chia and reopening doesn't work. Sometimes forcing a reload will let the computer see the plots again. The biggest issue is that while my computer can see and access the network drives when chia claims there are no plot folders added Chia won't finish a plot. It stops at 100% and keeps that plot as a temp file and won't even attempt to copy the file to the destination folder.
<img width="984" alt="Chia plots" src="https://user-images.githubusercontent.com/87924633/126912214-ebc9cf5a-e680-4e21-971f-433e06099ef8.png">

**Comment by keliew** [comment]
Start isolating the potential problems...

Could just be as simple as having different farmer keys.
